src/Features/CSharp/Portable/ExtractMethod/CSharpMethodExtractor.CSharpCodeGenerator.cs

```diff
@@ -270,39 +278,36 @@ private static SyntaxKind GetParameterRefSyntaxKind(ParameterBehavior parameterB
                                 SyntaxKind.OutKeyword : SyntaxKind.None;
             }
 
-            private OperationStatus<ImmutableArray<SyntaxNode>> CreateMethodBody(CancellationToken cancellationToken)
+            private ImmutableArray<StatementSyntax> CreateMethodBody(
+                SyntaxNode insertionPoint, CancellationToken cancellationToken)
             {
                 var statements = GetInitialStatementsForMethodDefinitions();
 
-                statements = SplitOrMoveDeclarationIntoMethodDefinition(statements, cancellationToken);
+                statements = SplitOrMoveDeclarationIntoMethodDefinition(insertionPoint, statements, cancellationToken);
                 statements = MoveDeclarationOutFromMethodDefinition(statements, cancellationToken);
                 statements = AppendReturnStatementIfNeeded(statements);
                 statements = CleanupCode(statements);
 
-                // set output so that we can use it in negative preview
-                var wrapped = WrapInCheckStatementIfNeeded(statements);
-                return CheckActiveStatements(statements).With(wrapped.ToImmutableArray<SyntaxNode>());
+                return statements;
             }
 
-            private IEnumerable<StatementSyntax> WrapInCheckStatementIfNeeded(IEnumerable<StatementSyntax> statements)
+            private ImmutableArray<StatementSyntax> WrapInCheckStatementIfNeeded(ImmutableArray<StatementSyntax> statements)
             {
                 var kind = CSharpSelectionResult.UnderCheckedStatementContext();
                 if (kind == SyntaxKind.None)
-                {
                     return statements;
-                }
 
-                if (statements.Skip(1).Any())
+                if (statements.Length != 1)
```

Comment:
this is intentionally not the same as before.  the 0 and >=2  cases were handled the same way in the prior code in two separate branches.  i merged them into one branch here.